qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -50,3 +50,14 @@
                     # support RHEL.7 guest or later
                     no RHEL.6
                     flags = "smap"
+                - test_dies:
+                    # support RHEL.8.1.0 host or later
+                    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0
+                    # support RHEL.7.8 guest or later
+                    no RHEL.6
```

Comment:
Support RHEL.7.8 or later, so here should `no RHEL.7.0...RHEL.7.7`, right?

Reply:
> Support RHEL.7.8 or later, so here should `no RHEL.7.0...RHEL.7.7`, right?

Yes, update and thanks.